pkg/ovs/daemonset.go

```diff
@@ -0,0 +1,205 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package ovs
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/openstack-k8s-operators/lib-common/modules/common/env"
+	"github.com/openstack-k8s-operators/ovs-operator/api/v1beta1"
+
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func getNetworksList(
+	instance *v1beta1.OVS,
+) string {
+	networks := "["
+	for physNet := range instance.Spec.NicMappings {
+		networks += fmt.Sprintf(
+			`{"name": "%s", "namespace": "%s"},`,
+			physNet, instance.Namespace,
+		)
+	}
+	networks = strings.TrimSuffix(networks, ",")
+	networks += "]"
+	return networks
+}
+
+func getPhysicalNetworks(
+	instance *v1beta1.OVS,
+) string {
+	physNets := ""
+	for physNet := range instance.Spec.NicMappings {
+		// NOTE(slaweq): to make things easier, each physical bridge will have
+		//               the same name as "br-<physical network>"
+		// NOTE(slaweq): interface names aren't important as inside Pod they will have
+		//               names like "net1, net2..." so only order is important really
+		physNets += fmt.Sprintf("%s ", physNet)
+	}
+	physNets = strings.TrimSuffix(physNets, ",")
```

Comment:
you could just create an array of all physnets and use strings.Join(). to get the list its possible to use maps.Keys)().  https://go.dev/play/p/A88LqdzOg97?v=goprev

Reply:
Done, thx